README.md

```diff
@@ -20,4 +20,4 @@ Get in touch with one of the organisers to get access to the repository :smiley:
 
 ## Getting approval access
 Want to help us on the website? We'd love you to help!
-Contact one of the Fenders Organisers and we can discuss adding you as a maintainer on the website 
\ No newline at end of file
+Contact one of the Fenders Organisers and we can discuss adding you as a maintainer on the website
```

Comment:
This should probably be done as part of another PR and most likely automatically done by your editor?

Reply:
Didn't actually add that. I'm guessing VSCode added a new line automatically which highlighted it. Probably from above when I ticked the checkbox